### ISSUE TITLE
[Bug]: yt-dlp package claims to be version 2023.10.13 while the program itself reports to be the previous stable version, namely 2023.03.04.

### ISSUE THREAD

**Issue by mykey** [ScoopInstaller/Main#5379]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

yt-dlp

### Expected/Current Behaviour

I have a pretty low priority bug with yt-dlp: The yt-dlp package claims to be version 2023.10.13, while the program itself reports to be the previous stable version, namely version 2023.03.04. 

The latter version works fine though, just wanted to give a heads-up. Thanks for all the good work.

### Steps to Reproduce

```console
PS> scoop info yt-dlp

Name        : yt-dlp
Description : A youtube-dl fork with additional features and fixes.
Version     : 2023.10.13
Bucket      : main
Website     : https://github.com/yt-dlp/yt-dlp
License     : Unlicense
Updated at  : 14/10/2023 02:33:06
Updated by  : github-actions[bot]
Installed   : 2023.10.13
Binaries    : yt-dlp.exe
Suggestions : ffmpeg

PS> yt-dlp --version
2023.03.04

PS> yt-dlp -U
Available version: stable@2023.10.13, Current version: stable@2023.03.04
ERROR: You installed yt-dlp with pip or using the wheel from PyPi; Use that to update
```

### Possible Solution

Either the package meta-information has to be updated in the manifest to reflect the correct version 2023.03.04 or the newest stable version, v 2023.10.13, has to be included.

### Scoop and Buckets Version

```console
PS> scoop --version
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
5865beb55 (HEAD -> master, origin/master, origin/HEAD) sidekick-browser: Update to version 116.53.1.36304-816141b

'main' bucket:
0e8a4c888 (HEAD -> master, origin/master, origin/HEAD) trivy: Update to version 0.47.0

'nerd-fonts' bucket:
d9fd1a7 (HEAD -> master, origin/master, origin/HEAD) SarasaGothic: Update to version 0.42.5

'versions' bucket:
8e1aa31fd (HEAD -> master, origin/master, origin/HEAD) vscode-insiders: Update to version 1.85.0-insider+1699249004222
```

### Scoop Config

```console
PS> scoop config


aria2-warning-enabled : False
last_update           : 2023-11-06T10:48:42.1097470+01:00
aria2-enabled         : False
scoop_branch          : master
scoop_repo            : https://github.com/lukesampson/scoop
```

### PowerShell Version

```console
PS> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.19041.3570
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.3570
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by HUMORCE** [comment]
```
PS> yt-dlp --version
2023.03.04

PS> yt-dlp -U
Available version: stable@2023.10.13, Current version: stable@2023.03.04
ERROR: You installed yt-dlp with pip or using the wheel from PyPi; Use that to update
```

you have another yt-dlp installation via pip.

check:

```
scoop which yt-dlp
where.exe yt-dlp
```

**Comment by HUMORCE** [comment]
Temporarily closed, feel free to leave a comment below.

**Reasons:**
- Cannot reproduce.
- No response and additional information
